vnext/ReactWindows-Desktop.sln

```diff
@@ -128,7 +128,6 @@ Global
 		Mso\Mso.vcxitems*{1958ceaa-fbe0-44e3-8a99-90ad85531ffe}*SharedItemsImports = 4
 		Shared\Shared.vcxitems*{2049dbe9-8d13-42c9-ae4b-413ae38fffd0}*SharedItemsImports = 9
 		Chakra\Chakra.vcxitems*{6f354505-fe3a-4bd2-a9a6-d12bbf37a85c}*SharedItemsImports = 4
-		Shared\Shared.vcxitems*{6f354505-fe3a-4bd2-a9a6-d12bbf37a85c}*SharedItemsImports = 4
```

Comment:
Please revert. Removing a VCXITEMS project from a solution does nothing in terms of dependency management, and only hides the sources from Visual Studio.

Note, MSBuild does not use the solution for other purpose than listing the top-level projects. Dependencies are resolved within the .*proj files.

Reply:
Noticed the duplicate removal. Resolving.